dockerfiles/Dockerfile.webpack

```diff
@@ -0,0 +1,5 @@
+FROM node:16
```

Comment:
I thought we changed this to a more modern node? Not sure where that change was tho, seems like it would apply here?